src/arch.rs

```diff
@@ -255,3 +286,144 @@ pub enum SingleStepGdbBehavior {
     #[doc(hidden)]
     Unknown,
 }
+
+/// (LLDB extension) Describes the register info for a single register of the
```

Comment:
please hide these new types behind a `pub mod lldb` module to avoid cluttering the `arch` module with a bunch of new lldb-specific types.